jet/files/src/main/java/com/hazelcast/samples/jet/files/unifiedapi/Trade.java

```diff
@@ -32,10 +32,11 @@
  */
 public class Trade implements Serializable {
 
-    private long time;
-    private String ticker;
-    private long quantity;
-    private long price;
+    // they need to be public since they are accessed from field mappers
```

Comment:
You could create setters for these instead, would remove the need for the checkstyle exception.